core/src/kvs/lq_structs.rs

```diff
@@ -0,0 +1,126 @@
+use crate::dbs::node::Timestamp;
```

Comment:
Just a nit, can we rename to something other than `lq_structs` to `lq.rs` or `lq_helpers.rs` or anything else that you deem suitable for clarity

Reply:
We already have `lq.rs` in a few places, so I think we could avoid that. `lq_helpers` I think implies functions. This is more about just carrier structures. They do have impl associated with them, but otherwise it's just a single place to find a relevant structure for the live query information chain.

I still think lq_structs is the preferred name, but I am open to suggestions still. And would follow up in separate PR for that - thanks for not making it a blocker!